prow/plugins/lgtm/lgtm.go

```diff
@@ -263,35 +266,68 @@ func handle(wantLGTM bool, config *plugins.Configuration, ownersClient repoowner
 	// Only add the label if it doesn't have it, and vice versa.
 	labels, err := gc.GetIssueLabels(org, repoName, number)
 	if err != nil {
-		log.WithError(err).Errorf("Failed to get the labels on %s/%s#%d.", org, repoName, number)
+		log.WithError(err).Errorf("Failed to get issue labels.")
 	}
 	hasLGTM := github.HasLabel(LGTMLabel, labels)
 
 	// remove the label if necessary, we're done after this
 	if hasLGTM && !wantLGTM {
 		log.Info("Removing LGTM label.")
-		return gc.RemoveLabel(org, repoName, number, LGTMLabel)
-	}
-
-	// add the label if necessary, we're done after this
-	if !hasLGTM && wantLGTM {
+		if err := gc.RemoveLabel(org, repoName, number, LGTMLabel); err != nil {
+			return err
+		}
+		botname, err := gc.BotName()
```

Comment:
We can skip trying to clean up the "LGTM added." comment in repos where `StoreTreeHash` is not enabled. This will save API tokens that would be used to list the issue comments.
If we fail to delete a few comments after a config change disables the feature in a repo it isn't a big deal.